packages/appium/lib/main.js

```diff
@@ -398,7 +398,7 @@ export {main, init, resolveAppiumHome};
 
 /**
  * @typedef ServerInitData
- * @property {AppiumDriver} appiumDriver - The Appium driver
+ * @property {import('./appium').AppiumDriver} appiumDriver - The Appium driver
  * @property {import('appium/types').ParsedArgs} parsedArgs - The parsed arguments
```

Comment:
almost everywhere else you updated this to `@appium/types`, is there a reason you didn't here?